index.html

```diff
@@ -44,129 +50,172 @@ <h1>I'm Abigael <br> Glad to see <br> you</h1>
+                <h2 class="desktop-only">Uber Navigation</h2>
+
+                <div class="experience-section">
+                    <p class="mobile-only canopy">Canopy</p>
+                    <p class="desktop-only canopy">Uber</p>
+                    <img src="images/Counter.svg" alt="" aria-hidden="true">
+                    <p class="mobile-only bullet">Back End Dev</p>
+                    <p class="desktop-only bullet">Lead Developer</p>
+                    <img src="images/Counter.svg" alt="" aria-hidden="true">
+                    <p class="mobile-only bullet">2015</p>
+                    <p class="desktop-only bullet">2018</p>
+                </div>
+                <p class="card-text mobile-only">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="card-text desktop-only">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                <ul class="skills">
+                    <li>html</li>
+                    <li class="desktop-only">Ruby on rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button type="submit">See Project</button>
             </div>
-            <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="skills">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-            </ul>
-            <button type="submit">See Project</button>
+            
         </section>
     </div>
 
     <!-- About Me section  -->
-    <div class="about">
-        <section class="about-me">
-            <h2>About <br> Myself</h2>
-            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have 
-                a project you need coded, don’t hestiate to contact me.</p>
-            <h3>Lets Connect</h3>
-            <img src="images/social-media-icons.png" alt="" aria-hidden="true">
-            <button type="submit">Get my resume</button>
-        </section>
-    </div>
-
-    <!-- Languages Section -->
-    <div class="languages-section">
-        <section class="languages">
-            <div class="languages-title">
-                <h4>Languages</h4>
-                <img src="images/downarrow.svg" alt="">
-            </div>
-            
-            <div class="language-list">
-                <div class="language">
-                    <img src="images/js-icon.svg" alt="" aria-hidden="true">
-                    <p>JavaSript</p>
-                </div>
-                <div class="language">
-                    <img src="images/html-icon.svg" alt="" aria-hidden="true">
-                    <p>HTML</p>
+    <div class="about-me-section">
+        <div class="about">
```

Comment:
- [ ] In this section you have two blocks; The left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right
- Currently, each of your blocks takes up 50% please fix it 👍🏿  

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_109](https://user-images.githubusercontent.com/69638013/166308008-64053b4f-86f4-4fe1-83a5-a001570d7079.png) | ![Screenshot_108](https://user-images.githubusercontent.com/69638013/166308025-0771a536-fba2-4976-b1f6-f2fa2f8922e2.png)

Reply:
This has also been resolved. Thank you.